CIScripts/Checkout/Zuul.ps1

```diff
@@ -34,8 +32,16 @@ function Get-ZuulRepos {
     )
 
     $Job.Step("Cloning zuul repositories", {
-        Invoke-NativeCommand -ScriptBlock {
+        $Global:LASTEXITCODE = $null
+        $ErrorActionPreference = "Continue"
+        $ScriptBlock = {
```

Comment:
Why not simple:
zuul-cloner.exe @ZuulClonerOptions @ProjectList 2>&1 | Write-Host
?